Exercise_1/day1/Program10.java

```diff
@@ -0,0 +1,29 @@
+package day1;
+
+public class Program10 {
+	public static void main(String[] args) {
+
```

Comment:
Get the input from the user instead of initialising

Reply:
Updated